src/Tickets/Emails/Email/Failed_Order.php

```diff
@@ -182,14 +200,20 @@ public function get_settings_fields(): array {
 	 * @return string The email content.
 	 */
 	public function get_content( $args = [] ): string {
-		// @todo: Parse args, etc.
-		$context = ! empty( $args['context'] ) ? $args['context'] : [];
-
 		// @todo: We need to grab the proper information that's going to be sent as context.
+		$is_preview = ! empty( $args['is_preview'] );
```

Comment:
Would it be better to use `tribe_is_truthy()` here, in case a string is passed, like '0'?